src/Native/System.Security.Cryptography.Native/pal_cms.h

```diff
@@ -61,3 +61,54 @@ extern "C" int CryptoNative_CmsGetRecipientStackFieldCount(CmsRecipientStack* re
 Shims the CMS_RecipientInfo_free method
 */
 extern "C" void CryptoNative_CmsRecipientDestroy(CMS_RecipientInfo* recipient);
+
+/*
+Initializes a CMS_ContentInfo of type EnvelopedData with the cipher given by the oid in algorithmOid
+and returns a pointer to it. 
+
+Status is set to 1 on success, 0 on unsupported algorithm, and -1 on invalid input.
```

Comment:
Ah, right CMS_EnvelopedData_create returns the pointer, not a success/failure code.  Okay, then.